app/src/androidTest/java/ch/epfl/sweng/swenggolf/NotificationsTest.java

```diff
@@ -166,6 +170,30 @@ public void levelUpNotificationWorks() {
         checkThatWeAreAt(ProfileActivity.class.getName(), R.id.name, user1.getUserName());
     }
 
+    @Test
+    public void getsNotificationWhenFriendPostsAnOffer() {
+        // write info that user2 follows user1 into the database
+        Database.getInstance().write(Database.FOLLOWERS_PATH, user2.getUserId(), user1.getUserId());
+        // make user1 create an offer
+        Config.setUser(user1);
+        activityTestRule.getActivity().getSupportFragmentManager().beginTransaction()
+                .replace(R.id.centralFragment, new CreateOfferActivity())
+                .commit();
+        String offerTitle = "This is an offer";
```

Comment:
I don't understand... Here we want the user to create an offer, so that it triggers a notification. It wouldn't work with an offer that was already created.